scalatest/src/main/scala/org/scalatest/tools/RunnerJFrame.scala

```diff
@@ -269,7 +269,7 @@ private[scalatest] class RunnerJFrame(
     reporterJPanel.add(eventsDetailsPanel, BorderLayout.CENTER)
     eventsJList.addListSelectionListener(
       new ListSelectionListener() {
-        def valueChanged(e: ListSelectionEvent): Unit = {
+        def valueChanged(e: ListSelectionEvent | Null): Unit = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.